### PR TITLE
Changelog for 0.9.0

### PR DESCRIPTION
**Pull request by jenshnielsen**
In preparation to cut 0.9.0 release candidate asap

### CODE REVIEW

**Comment by jenshnielsen** [comment]
This includes changes from https://github.com/QCoDeS/Qcodes/pull/1850/files that should land first

File: docs/changes/0.9.0.rst [hunk 1]
```diff
@@ -1,31 +1,40 @@
...
 Improved:
 _________
 
-* ...
-
+* Improvements to dataset notebooks (#1813)
+* Fix warning in matplotlib plotting for legacy dataset (#1839)
+* Add electrical delay parameter and function set_electrical_delay_auto for ZNB driver (#1849)
+* Keysight 344xxA: Fix formatting issues (#1851)
+* Fix docstrings in parameter.py #1855
```
Comment (astafan8) [inline docs/changes/0.9.0.rst]:
```suggestion
* Fix docstrings in parameter.py (#1855)
```

File: docs/changes/0.9.0.rst [hunk 2]
```diff
@@ -1,31 +1,40 @@
-Changelog for QCoDeS 0.9.0
-==========================
+Changelog for QCoDeS 0.9.0rc1
+=============================
 
 (NOTE RELEASED YET) The December 2019 release of QCoDeS
```
Comment (astafan8) [inline docs/changes/0.9.0.rst]: do i understand it correctly that becuase it's an rc we don't remove this "note", right?
```suggestion
The December 2019 release of QCoDeS
```

Comment (jenshnielsen) [reply docs/changes/0.9.0.rst]: Yes since this goes on the homepage I want to leave this until just before pushing the release to avoid confusion

**Pull request merged**